src/pcp.rs

```diff
@@ -113,19 +113,22 @@ pub enum PcpError {
 
 /// A value of a certain type. Implementations of this trait specify an arithmetic circuit that
 /// determines whether a given value is valid.
-pub trait Value:
-    Sized
-    + PartialEq
-    + Eq
-    + Debug
-    + for<'a> TryFrom<(<Self as Value>::Param, &'a [Self::Field]), Error = PcpError>
-{
+pub trait Value: Sized + PartialEq + Eq + Debug {
```

Comment:
```suggestion
pub trait Value: Sized + Eq + Debug {
```
[`Eq` implies `PartialEq`](https://doc.rust-lang.org/std/cmp/trait.Eq.html)

Reply:
Done.